compiler/runtime/CodeCacheTypes.hpp

```diff
@@ -26,19 +26,23 @@
 #define __TPF_DO_NOT_MAP_ATOE_REMOVE
 #endif
 
+#undef CODECACHE_DEBUG
+
 #include <stddef.h>
 #include <stdint.h>
+#if defined(CODECACHE_DEBUG)
+#include <stdio.h>
+#endif /* CODECACHE_DEBUG */
+
 #include "runtime/MethodExceptionData.hpp"
 
 /*
  *  Debugging help
  */
-#ifdef CODECACHE_DEBUG
-#define mcc_printf if (CodeCacheDebug) printf
-#define mcc_hashprintf /*printf*/
+#if defined(CODECACHE_DEBUG)
+#define mcc_printf(...) printf(__VA_ARGS__)
```

Comment:
Hm... so in other parts of the compiler, we use `##__VA_ARGS__` (with the leading `##`). I'm not sure if that matters. I know the first is technically a non-standard compiler extensions but it seems to work on all our build compilers.

Reply:
using VA_ARGS is probably cleaner, and since it is supported, I'll push a version that uses it.